Feature/6

Atom098890 opened the pull request.

Commit:
Расчитал расстояние от текущего места до назначения

Commit:
add homework 5-languages
switch/case

Commit:
add homework 6-driver

korbutds commented on 4-navigator/homework.js:
```diff
@@ -0,0 +1,7 @@
+let positionLat = 6;
+let positionLong = 10;
+let addressLat = 8;
+let addressLong = 16;
+
+let destination = Math.sqrt(((addressLat - positionLat) ** 2) + ((addressLong - positionLong) ** 2))
```
Почему `let`? Ответьте, пожалуйста, ем отличается `let` от `const`?

Atom098890 replied:
const, это константа, её значение нельзя изменить

korbutds commented on 5-languages/homework.js:
```diff
@@ -0,0 +1,16 @@
+let languages = 'es';
```
Опять же, почему `let`?

korbutds commented on 5-languages/homework.js:
```diff
@@ -0,0 +1,16 @@
+let languages = 'es';
+
+switch (languages) {
+    case 'ru':
+        console.log('Привет!');
+        break;
+    case 'de':
+        console.log('Gutten tag!');
+        break;
+    case 'es':
+        console.log('Hola!');
+        break;
+    default:
+        console.log("I don't know this languages!");
+}
```
Для практики, попробуйте переписать на `switch (true) {...}`

korbutds commented on 6-driver/homework.js:
```diff
@@ -0,0 +1,6 @@
+const age = 19;
+const isDrink = false;
+
+const isDriver = age > 18 && hasLicence && !isDrink;
```
А правильно ли работает логика? Что будет если пользователю 18 лет?

Commit:
Правка с let на const в файле 4-navigator
Переписал switch в файле 5-languages
Правка условия в файле 6-driver, добавил больше либо равно 18

Commit:
Правка с let на const в файле 4-navigator
Переписал switch  в файле 5-languages
Правка в файле 6-diver, добавил больше либо равно 18

korbutds reviewed (approved):
Просьба отвечать на каждый комментарий, чтобы было понятно, что вопрос понятен или принят к работе.
